### PR TITLE
Proposal for Version 5

### PR DESCRIPTION
**Pull request by mploed**
I have recently worked a lot with a modified version of the bounded context design canvas and made a lot of good expereiences with the structure which I would like to propose as Version 5.

**Key changes:**

- Renamed Description to Purpose as already discussed in Issue #24 
- Added a section for Assumptions: Every design is based on assumptions and it is a good idea to make them explicit
- Added a section for Metrics: Recently I often defined metrics with teams which help them to verify / validate their chosen context boundaries. Thinking in metrics also often helped to unlock additional insights
- Added a section for Questions: In every design session there will be open questions and it's good to explicitly document them

This pull request adds:

- Templates for drawio
- HTML Template
- JPEG Templates

This pull request currently does not contain:

- Updates to the French version
- An update of the example

Let me know what you think.

### CODE REVIEW

**Comment by Max-Git** [comment]
This is a great improvement, thanks @mploed!
There's a typo here: "Most design happes based on assumptions" (*happens)
And maybe that would be great to have an updated version of the filled in example at the end. 
Tbh I'd like to have more examples about the metrics section. I have the feeling that it's a nice addition to the canvas because we speak a lot about metrics at this stage but I'm not 100% sure about what you'd like to see there.
Thanks!

File: README.md [hunk 1]
```diff
@@ -103,6 +105,22 @@ What are the key domain terms that exist within this context, and what do they m
 ### Business Decisions
 What are the key business rules and policies within this context?
 
+### Assumptions
+You will never make design decisions having a full knowledge about everything in your domain. Most design happes based on assumptions and it is highly recommended to make them explicit. This can be done in this section of the Bounded Context Design Canvas.
+
+### Verification Metrics 
+Domain Driven Design is about an iterative approach towards modelling and design based on continuous learning. Metrcis can help you
```
Comment (aafsicha) [inline README.md:112]: typo: Metrics

File: README.md [hunk 2]
```diff
@@ -103,6 +105,22 @@ What are the key domain terms that exist within this context, and what do they m
...
+gathering valuable input for those learnings (think about build-measure-learn). Think about mertics that you and your team can define in order to gather learnings if the chosen boundaries of your bounded context are a good fit or not.
```
Comment (aafsicha) [inline README.md:113]: typo: metrics

**Comment by mploed** [comment]
@Max-Git example is updated

**Review by Max-Git — APPROVED** [review]
Very good improvements of the canvas. Thanks @mploed

**Review by NTCoding — APPROVED** [review]

**Review by yellowbrickc — APPROVED** [review]
I love it! I always have an Open questions section, which I maintained (until now) on the side. The other two sections are very important too, it is a great improvement.

**Comment by yellowbrickc** [comment]
I will add two issues if you agree: one for @NTCoding for the miro template (nobody else can do that, I assume) and one for or the french translation (maybe @Max-Git or @aafsicha can have a fast run at it?).

**Pull request merged**